test/splitgraph/commands/test_es_aggregation_pushdown.py

```diff
@@ -112,7 +180,50 @@ def test_elasticsearch_gropuing_clauses_only(snapshot, local_engine_empty):
 
 
 @pytest.mark.mounting
-def test_elasticsearch_grouping_and_aggregations_bare(snapshot, local_engine_empty):
+def test_simple_grouping_clauses_filtering(snapshot, local_engine_empty):
+    _mount_elasticsearch()
+
+    # Single column grouping
+    query = "SELECT state, gender FROM es.account WHERE state IN ('TX', 'WA', 'CO') GROUP BY state, gender"
+
+    # Ensure grouping is going to be pushed down
+    result = get_engine().run_sql("EXPLAIN " + query)
+    assert _extract_queries_from_explain(result)[0] == {
+        "query": {
+            "bool": {
+                "must": [
+                    {
+                        "bool": {
+                            "should": [
+                                {"term": {"state": "TX"}},
+                                {"term": {"state": "WA"}},
+                                {"term": {"state": "CO"}},
+                            ]
+                        }
+                    }
+                ]
+            }
+        },
+        "aggs": {
+            "group_buckets": {
+                "composite": {
+                    "sources": [
+                        {"state": {"terms": {"field": "state"}}},
+                        {"gender": {"terms": {"field": "gender"}}},
+                    ],
+                    "size": 5,
```

Comment:
Is this the scroll size set in https://github.com/splitgraph/postgres-elasticsearch-fdw/blob/249903034e348e472fa46881c8def7217d70f5fe/pg_es_fdw/__init__.py#L265 ? How does it become so small?

Reply:
That is correct - I [deliberately set it to a small value](https://github.com/splitgraph/splitgraph/blob/multicorn-agg-pushdown-v2-cu-1z461e4/test/splitgraph/conftest.py#L153), so that the [pagination functionality](https://github.com/splitgraph/postgres-elasticsearch-fdw/blob/es-agg-pushdown-v2-cu-1z461e4/pg_es_fdw/__init__.py#L350) gets properly exercised.

In fact I was thinking of making parametrizing this property (e.g. use 1,5,10,100 for each test), but figured it would be an overkill.